src/actions/index.js

```diff
@@ -13,9 +13,9 @@ export const ACTION_TYPES = {
 };
 
 // This handles loading photos on mount. Temporary.
-export function getPhotos() {
+export function getPhotos(params, endpoint = '') {
   return dispatch => {
-    get('/photos', {feature: 'popular', image_size: [1, 200], tags: 1})
+    get(`/photos${endpoint}`, params)
```

Comment:
That was the hope anyway. You can pass in additional strings to build on the endpoint and pass in an object with the params you need.